### ISSUE TITLE
Failure to build libgraphics

### ISSUE THREAD

**Issue by scottwilliambeasley** [ace13/flatpak-dwarffortress#7]
OS: Ubuntu 17.10
Arch: x86_64
Issue: Failure to build libgraphics module
Relevant Output of `make`:
```
========================================================================
Building module libgraphics in /home/scott/Git/flatpak-dwarffortress/.flatpak-builder/build/libgraphics-1
========================================================================
bash: flatpak: command not found
flatpak --user remote-add --if-not-exists flathub --from https://flathub.org/repo/flathub.flatpakrepo
make: *** No rule to make target 'com.bay12games.DwarfFortress.json.in', needed by 'com.bay12games.DwarfFortress.json'.  Stop.
make: *** Waiting for unfinished jobs....
bash: flatpak: command not found
make: *** [Makefile:58: deps] Error 127
Error: module libgraphics: Child process exited with code 2
Makefile:69: recipe for target 'build' failed
make: *** [build] Error 1
```

Output of `which flatpak`:

```
scott@black-lotus:~/Git/flatpak-dwarffortress$ which flatpak
/usr/bin/flatpak
```

**Comment by ananace** [comment]
It looks like it's using the wrong makefile, I'm guessing your flatpak version is pre 0.9.2?

**Comment by ananace** [comment]
Pushed a commit that should fix the issue for you

**Comment by scottwilliambeasley** [comment]
I've received the ff error when building the dwarf-fortress module:

```
Committing stage cleanup to cache
Finishing app
error: Unknown option --extension=com.bay12games.DwarfFortress.dfhack=directory=/app/dfhack
Error: Child process exited with code 1
Makefile:69: recipe for target 'build' failed
make: *** [build] Error 1
```

You are right - the version of flatpak that Ubuntu 17.10 currently provides in the default repos is 0.8.9.

Upon further investigation, I see that this version of flatpak did not yet support the `--extension` flag in the `flatpak build-finish` command, which your build currently relies on.

It certainly seems like an uphill battle attempting to debug every single message when using this version of flatpak. 

I installed the third party repo instead to get a much more updated version of flatpak, and now it works fine.

My recommendation would be to simply drop support for older versions of flatpak and instead append to the README.md that version 0.9.2 or later is required, followed by indicating this may need a third party repo (since some distributions haven't included it yet). I submitted a pull for this.
